Update Timestamp Autocorrection and Historical Data Ingestion Documen… (#5256)
SumoLogic/sumologic-documentation: docs/send-data/collector-faq.md

See [using _format for troubleshooting](/docs/send-data/reference-information/time-reference.md) timestamps.

#### Assumption: Data is less than 30 days but within 365 days

* To ingest historical data older than 30 days but within 365 days, you must specify a `timestamp` field using a regex locator and a valid date format.

#### Assumption: Data is older than 365 days

Sumo Logic assumes that all log message times fall within a window of -1 year through +2 days compared to the current time. Any log messages with a parsed timestamp outside of that window are automatically re-stamped with the current time.
* Data older than 365 days can still be ingested. However, even if a custom timestamp is provided, it will be autocorrected to the current time unless technical support disables this function at the organization level.

> * To ingest data older than 365 days with the original timestamp intact, you must contact technical support to disable the autocorrection function at the org level.

Reviewer: ```suggestion
* To ingest data older than 365 days with the original timestamp intact, you'll need to contact contact [Sumo Logic Support](https://support.sumologic.com/support/s) to disable the autocorrection function at the org level.
```
